### DISCUSSION TITLE
Livarno Smart Led Filament not regonized

### DISCUSSION THREAD

**Discussion by Theo61** [Koenkk/zigbee2mqtt#14511] [Devices]
Hallo LED light Livarno Smart Led Filament  E27 is not regonized. In the log it has modelID: 'TS0501A', manufacturerName: '_TZ3000_nosnx7im'. This is already known in the Lidl.js, but with a different Model ID. The ModelID is HG08134A/HG08134B I ried to make external device, d\but that failed aldo.

**Comment by Koenkk** [comment]
`_TZ3000_nosnx7im` is supported in z2m 1.28.0
